doc/development_setup.md

```diff
@@ -26,6 +26,43 @@ Rust formatting can be achieved by adding the following to settings.json in VSCo
  - Frontend [npm](https://www.npmjs.com/)
  - Backend [cargo](https://crates.io/)
 
+### Installing node + npm
+
+If you are using unix, macOS, and windows WSL, consider using [nvm](https://github.com/nvm-sh/nvm) to manage your node environment:
+
+```bash
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.3/install.sh | bash
+nvm install --lts
+nvm use --lts
+```
+
+Or if you use the default macOS starting with Catalina shell `zsh`, try:
+
+```zsh
+sh -c "$(curl -fsSL https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.3/install.sh)"
+```
+
+### Installing rust + cargo
+
+If you’re using Linux or macOS, open a terminal and enter the following command:
+
+```bash
+$ curl --proto '=https' --tlsv1.2 https://sh.rustup.rs -sSf | sh
```

Comment:
This maybe does now the right thing, but it won't if Rust gets upgraded in future. In backend README we defined we want to use Rust 1.67.1, so please install this version.